src/ts/core/srs/srs.ts

```diff
@@ -44,3 +47,49 @@ export function triggerNextBucket() {
             )
     )
 }
+
+const intervalStyle = (interval: string, nextReviewTip: string) => {
+    return `
+        [data-link-title^="[[interval]]:${interval}"] ~ [data-link-title]:last-child::before {
+            content: "review interval: ${nextReviewTip}";
+        }
+    `
+}
+
+const easeStyle = (ease: string, darkColor: string, lightColor: string) => {
+    return `
+        [data-link-title^="[[factor]]:${ease}"] ~ [data-link-title]:last-child::before {
+            border: 1px solid ${darkColor};
+            color: ${darkColor};
+            background: ${lightColor};
+        }
+    `
+}
+
+injectStyle(
+    `
+    [data-link-title^="[[interval]]:"], [data-link-title^="[[factor]]:"] {
+        display: none;
+    }
+    [data-link-title^="[[factor]]:"] ~ [data-link-title]:last-child::before {
+        font-style: italic;
+        margin-right: 6px;
+        padding: 3px 6px;
+        border-radius: 12px;
+        font-size: smaller;
+        white-space: nowrap;
+    }
+    ${intervalStyle('', '1+ month')}
+    ${intervalStyle('1.', '1 day')}
+    ${range(2, 6)
+        .map(days => intervalStyle(`${days}.`, `${days} days`))
+        .join('\n')}
+    ${range(7, 30)
+        .map(days => intervalStyle(`${days}.`, `${Math.floor(days / 7)} weeks`))
```

Comment:
I'd actually like to see the fractions here (to a first decimal) (same for months)